### PR TITLE
We should lint the swift

### PR DESCRIPTION
**Pull request by djensenius**
This mostly Swift code cleanup 
I added a Swift linter. Linting is good. It keeps our code consistent and me happy.
I also updated the outdated cocoa pods which eliminated most outside warnings (yay!), but introduced some warnings with `Tabman` (grrr).

### CODE REVIEW

**Comment by djensenius** [comment]
Looks like this is failing CI because the CocoaPods in travis-ci are not up to date? Not sure how to fix this exactly, and gotta step out now. I can investigate later, unless you have any good ideas @frogg. Otherwise this PR is ready for review / merge.

**Comment by djensenius** [comment]
Looks like I fixed it. :-)

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 language: objective-c
 osx_image: xcode9.3
+before_install:
+- pod repo update
```
Comment (frogg) [inline .travis.yml]: Hehe yep, now that the pods are not part of the repo anymore we have to update them manually, of course.

**Comment by djensenius** [comment]
@frogg is this PR okay to merge?

**Comment by frogg** [comment]
@djensenius This looks good to me at a first glance but I want to run it locally before merging. Will do that tonight!

**Pull request merged**